src/lucky/action_pipes.cr

```diff
@@ -178,4 +182,20 @@ module Lucky::ActionPipes
   def continue : Lucky::ActionPipes::Continue
     Lucky::ActionPipes::Continue.new
   end
+
+  def publish_before_event(pipe_name : String, continued : Bool)
```

Comment:
These methods should probably be private right?

Reply:
Good catch